website/docs/secrets/bootstraping-secrets.mdx

```diff
@@ -2,20 +2,151 @@
 title: Bootstrapping Secrets
 hide_title: true
 ---
+import AlphaWarning from "../_components/_alpha_warning.mdx";
 
 # Overview
 
-// State what we mean by bootstrapping secrets (Provide needed secrets for Profiles Helm Repo, Profiles, other applications needed in the cluster provisioning cluster)
-// State the 2 possible ways to do this (SecretSync, Terraform Modules)
+When accessing protected resources there is a need for a client to authenticate before
+the access is granted and the resource is consumed. For authentication, a client presents
+credentials that are either created manually or available through infrastructure. A common scenario
+is to have a secrets store.
+
+Weave Gitops allows you to provision the secret management infrastructure as part of [its capabilities](./setup-secrets-operator.mdx).
+However, in order to provision, as any other application that has secrets, we need to create the secret needed for installing it.
+This is known as a chicken-egg scenario that get addressed by providing an initial secret. This secret we call it
+bootstrapping secret.
+
+Bootstrapping secrets will be, not only present while provision your secrets management solution, but in
```

Comment:
I still find this whole paragrap a bit complex and hard to follow. If we are trying to tell the user that the `bootstrapping secrets` functionally is beneficial for other things as well, then we can say something like:
`Bootstrapping secrets comes in handy, not only while provisioning your secrets management solution, but also in any platform provisioning task where the existence of the secret is a prerequisite.  Another common example could be provisioning
platform capabilities via [profiles](../../../cluster-management/getting-started/#profiles-and-clusters)
that are stored in [private repositories](https://fluxcd.io/flux/guides/helmreleases/#helm-repository-authentication-with-credentials).`